### PR TITLE
Rename shouldLockout

### PR DESCRIPTION
**Pull request by HaoK**
New PR from release branch

cc @divega

### CODE REVIEW

File: src/Microsoft.AspNet.Identity/LockoutOptions.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ public class LockoutOptions
         /// <summary>
         ///     If true, will enable user lockout when users are created
         /// </summary>
-        public bool EnabledByDefault { get; set; } = false;
+        public bool AllowedForNewUsers { get; set; } = false;
```
Comment (divega) [inline src/Microsoft.AspNet.Identity/LockoutOptions.cs]: It seems I am a bit lost with one thing here: aren't we supposed to flip the default for this? If yes, where is that flip happening?

Comment (HaoK) [inline src/Microsoft.AspNet.Identity/LockoutOptions.cs]: In a local change that I forgot to push :) Was lost in the merge from the dev branch...

Comment (HaoK) [inline src/Microsoft.AspNet.Identity/LockoutOptions.cs]: Pushed now

**Comment by divega** [comment]
:shipit:

**Comment by rustd** [comment]
@HaoK can you please update the templates repo as well. This is in the release branch.
https://github.com/aspnet/Templates/blob/release/src/Rules/StarterWeb/IndividualAuth/Controllers/AccountController.cs#L66

**Comment by HaoK** [comment]
1ed2ebcd962cf86c57da9e54626732c9feb0d21c

**Pull request closed without merging**